Sources/NestKit/GitHub/GitHubURLBuilder.swift

```diff
@@ -24,6 +24,8 @@ public enum GitHubURLBuilder {
         switch version {
         case .latestRelease:
             return baseURL.appending(components: "repos", owner, repository, "releases", "latest")
+        case .latestAvailableRelease:
+            return baseURL.appending(components: "repos", owner, repository, "releases")
```

Comment:
Since the latest version is returned in descending order, I think pagination support is not necessary for now.